semcore/data-table/src/Body.tsx

```diff
@@ -58,12 +59,59 @@ class Body extends Component<AsProps, State> {
   firstRowRef = React.createRef<HTMLDivElement>();
   firstRowResizeObserver: ResizeObserver | null = null;
 
+  lockedCell: [HTMLElement | null, boolean] = [null, false];
+
   getRowHeight = () => {
     const { virtualScroll } = this.asProps;
     const rowHeightFromProps = typeof virtualScroll === 'object' && virtualScroll?.rowHeight;
```

Comment:
Seems that navigation is broken in virtual scroll when the focused cell is unloaded from the viewport...

Reply:
yes, this will be fixed in the next versions